### PR TITLE
Pr140 - Site role Update

### PR DESCRIPTION
**Pull request by sbarguil**

### CODE REVIEW

File: yang/IETF-04/ietf-vpn-common.yang [hunk 1]
```diff
@@ -429,28 +459,64 @@ module ietf-vpn-common {
        where Hubs can act as Spoke for certain part of
        the network or Spokes as Hubs.";
   }
-  
+    
   identity role {
-   description
-     "Base identity for site or node role type.";
+    description
+    "Base identity for site type.";
```
Comment (boucadair) [inline yang/IETF-04/ietf-vpn-common.yang:465]:
```suggestion
    "Base identity for site or node type.";
```

File: yang/IETF-04/ietf-vpn-common.yang [hunk 2]
```diff
@@ -429,28 +459,64 @@ module ietf-vpn-common {
...
+   }
+
+   identity any-to-any-role {
+    base role;
+    description
+    "Site in an any-to-any IP VPN.";
```
Comment (boucadair) [inline yang/IETF-04/ietf-vpn-common.yang:471]:
```suggestion
    "Site or Node in an any-to-any IP VPN.";
```

File: yang/IETF-04/ietf-vpn-common.yang [hunk 3]
```diff
@@ -429,28 +459,64 @@ module ietf-vpn-common {
...
+   }
+
+   identity spoke-role {
+    base role;
+    description
+    "Spoke site in a Hub-and-Spoke IP VPN.";
```
Comment (boucadair) [inline yang/IETF-04/ietf-vpn-common.yang:477]:
```suggestion
    "Spoke site/node in a Hub-and-Spoke IP VPN.";
```

File: yang/IETF-04/ietf-vpn-common.yang [hunk 4]
```diff
@@ -429,28 +459,64 @@ module ietf-vpn-common {
...
+   }
+
+   identity hub-role {
+    base role;
+    description
+    "Hub site in a Hub-and-Spoke IP VPN.";
```
Comment (boucadair) [inline yang/IETF-04/ietf-vpn-common.yang:483]:
```suggestion
    "Hub site/node in a Hub-and-Spoke IP VPN.";
```

File: yang/IETF-04/ietf-vpn-common.yang [hunk 5]
```diff
@@ -429,29 +429,35 @@ module ietf-vpn-common {
...
-  }
-  
-  identity any-to-any-role {
-   base role;
-   description
-     "Site/Node in an any-to-any VPN.";
-  }
-  
-  identity spoke-role {
+    description
```
Comment (boucadair) [inline yang/IETF-04/ietf-vpn-common.yang:464]: Role applies for both sites and nodes. Please maintain the OLD wording. Thanks

File: yang/IETF-04/ietf-vpn-common.yang [hunk 6]
```diff
@@ -429,29 +429,35 @@ module ietf-vpn-common {
...
+   }
+
+   identity any-to-any-role {
     base role;
     description
-      "Spoke site/node in a Hub-and-Spoke IP VPN.";
-  }
-  
-  identity hub-role {
+    "Site in an any-to-any IP VPN.";
```
Comment (boucadair) [inline yang/IETF-04/ietf-vpn-common.yang]: I suggest to maintain the initial wording

File: yang/IETF-04/ietf-vpn-common.yang [hunk 7]
```diff
@@ -429,29 +429,35 @@ module ietf-vpn-common {
...
+   }
+
+   identity spoke-role {
     base role;
     description
-      "Hub site/node in a Hub-and-Spoke VPN.";
-  }
+    "Spoke site in a Hub-and-Spoke IP VPN.";
```
Comment (boucadair) [inline yang/IETF-04/ietf-vpn-common.yang]: Please maintain the initial wording. Thanks

File: yang/IETF-04/ietf-vpn-common.yang [hunk 8]
```diff
@@ -429,29 +429,35 @@ module ietf-vpn-common {
...
+   identity hub-role {
+    base role;
+    description
+    "Hub site in a Hub-and-Spoke IP VPN.";
+   }
+
+  identity custom-role {
+    base role;
+    description
+    "Hub site in a Hub-and-Spoke IP VPN.";
```
Comment (boucadair) [inline yang/IETF-04/ietf-vpn-common.yang:489]: We need to provide more accurate description. Custom is not "clear"

File: yang/IETF-04/ietf-vpn-common.yang [hunk 9]
```diff
@@ -429,29 +429,35 @@ module ietf-vpn-common {
...
     description
-      "Hub site/node in a Hub-and-Spoke VPN.";
-  }
+    "Spoke site in a Hub-and-Spoke IP VPN.";
+   }
+
+   identity hub-role {
+    base role;
+    description
+    "Hub site in a Hub-and-Spoke IP VPN.";
```
Comment (boucadair) [inline yang/IETF-04/ietf-vpn-common.yang]: Please mention "node" as we are also using it for things such as:

              +--rw node-role?                 identityref

Comment (sbarguil) [reply yang/IETF-04/ietf-vpn-common.yang]: ae785b919c19f6a03564c9529fa39b0495983d48

**Pull request merged**